### DISCUSSION TITLE
Welcome to Not-so-Epic Discussions!

### DISCUSSION THREAD

**Discussion by hoover900** [hoover900/Not-so-Epic#55] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
I was wondering if you could make a theme for the ports emulator and the PC emulator

**Reply by ghost** [reply]
I currently am using this ES theme but it has no images or display when I scroll to the Ports emulator
